### ISSUE TITLE
[BUG] missing aria-label for list

### ISSUE THREAD

**Issue by tfl-jyskebank-dk** [kirbydesign/designsystem#3956]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When we're trying to add a [attr.aria-label] or aria-label to a 'kirby-list' this will add the label to a group surrounding the list. We have tried with VO. 

But if we add role="list" to the kirby-list and role="listitem" to the underlying child kirby-items. Then the name appears on the kirby-list. But there also appears an extra list inside the kirby-list without an accessible name when we check it with VO. 

## Describe how to reproduce the bug

Try creating a kirby-list and adding an aria-label. 

Then try testing it with a screenreader fx VoiceOver. 

```
<kirby-list [items]="infoLinks" aria-label="Liste med links">
    <kirby-item *kirbyListItemTemplate="let item"> {{ item.text }} </kirby-item>
</kirby-list>
```
## Which Kirby version was used?
10.3.0

## What was the expected behavior?
The expected behavior is that the list has an accessible name. 

## Add any screenshots
No screenshots provided. 

## Please complete the following information:
- OS: MacOs
- Browser: Chrome (Version 136.0.7103.114)
- Browser: Safari (Version 18.3.1 (20620.2.4.11.6))

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.
